src/Apps/Comparables/ComparablesView.tsx

```diff
@@ -0,0 +1,101 @@
+import { Box, Spinner } from "@artsy/palette"
```

Comment:
This is the actual main view, which will have a bunch of filter props passed in. It'll retrieve that set of filter results, and show the current filter in the UI (still todo on the UI). Then users can see the current filter, as well as start adjusting it and see new results.